controllers/grid/articleGalleys/form/ArticleGalleyForm.php

```diff
@@ -62,17 +62,15 @@ public function __construct($request, $submission, $publication, $articleGalley
         $this->addCheck(new \PKP\form\validation\FormValidatorCSRF($this));
 
         // Ensure a locale is provided and valid
-        $journal = $request->getJournal();
+        $locales = $submission->getPublicationLanguages($request->getJournal()->getSupportedSubmissionMetadataLocales(), $articleGalley?->getLanguages());
         $this->addCheck(
             new \PKP\form\validation\FormValidator(
                 $this,
                 'locale',
                 'required',
                 'editor.issues.galleyLocaleRequired'
             ),
-            function ($locale) use ($journal) {
```

Comment:
I think this is not correct: addCheck only accepts one parameter, so I think this would need to be FormValidatorCustom, that will then have this function as parameter.
Could you please change it?